recipe/patches/0010-Allow-overriding-CUDA-related-paths.patch

```diff
@@ -17,7 +17,7 @@ index ec9ae530..76d1fb9a 100644
  # We compute the rest based on those here to avoid re-searching and to avoid finding a possibly
  # different installation.
 -if(CMAKE_CUDA_COMPILER_TOOLKIT_ROOT)
-+if(FALSE)
++if(CMAKE_CUDA_COMPILER_TOOLKIT_ROOT AND NOT "$ENV{IN_PYTORCH_BUILD}" STREQUAL "1")
```

Comment:
FWIW conda-build sets the environment variable `CONDA_BUILD=1`

ref: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process

Reply:
Yeah, but that will be set in all the other downstream conda-forge builds as well, making it unusable to distinguish the situation between here (where we want to disable this part) and downstream (where we don't; at least not yet).

Reply:
Good point